### ISSUE TITLE
Update funding blurb

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1022]
# TL;DR
Need to update the footer text regarding funding of Lookit with new information!  Melissa should find out and provide the corresponding numbers for the new NSF grant. 

Existing text: This material is based upon work supported by the National Science Foundation (NSF) under Grants 1429216 and 1823919; the Center for Brains, Minds and Machines (CBMM), funded by NSF STC award CCF-1231216, and by an NSF Graduate Research Fellowship under Grant No. 1122374. Any opinion, findings, and conclusions or recommendations expressed in this material are those of the authors(s) and do not necessarily reflect the views of the National Science Foundation.

New text:

Existing text: This material is based upon work supported by the National Science Foundation (NSF) under Grants 1429216, 1823919, and 2209756; the Center for Brains, Minds and Machines (CBMM), funded by NSF STC award CCF-1231216, and by an NSF Graduate Research Fellowship under Grant No. 1122374. Any opinion, findings, and conclusions or recommendations expressed in this material are those of the authors(s) and do not necessarily reflect the views of the National Science Foundation.

[A new grant number has been added to the first list. It's very thrilling copy!]

**Comment by okaycj** [comment]
@mekline 

I would like to confirm that you're okay with updating translated text.

**Comment by mekline** [comment]
Yes!
